### PR TITLE
feat: add `array/base/broadcasted-ternary5d`

### LINKED ISSUE #3166: [RFC]: add `array/base/broadcasted-ternary5d`
### Description

This RFC proposes adding a new package to the `array/base/*` namespace: `@stdlib/array/base/broadcasted-ternary5d`.

This package is the ternary equivalent of [`@stdlib/array/base/broadcasted-binary5d`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/array/base/broadcasted-binary5d).

To provide a concrete example of what a PR adding the desired package should contain, see [`@stdlib/array/base/broadcasted-ternary2d`](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/array/base/broadcasted-ternary2d), which is the 2D package equivalent.

While the package proposed in this RFC will **not** match the implementations found in the above packages, those packages should provide a conceptual idea of what is desired. Do **not** simply copy-paste the code found in those packages.

### Notes

- If you are interested in contributing a PR which addresses this RFC and are a **first-time contributor** or not yet familiar with our project conventions, please do **not** submit LLM-generated code. Please consult our [contributing guidelines](https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md) and the associated [development guide](https://github.com/stdlib-js/stdlib/blob/develop/docs/development.md). Failure to respect project conventions will result in your PR being rejected without review. Thank you for understanding!

### Checklist

- [X] I have read and understood the [Code of Conduct](https://github.com/stdlib-js/stdlib/blob/develop/CODE_OF_CONDUCT.md).
- [X] Searched for existing issues and pull requests.
- [X] The issue name begins with `[RFC]:`.

### PR DESCRIPTION
**Pull request by Vinit-Pandit**
Resolves #3166

## Description

> What is the purpose of this pull request?

This pull request:

-   {{TODO: add description describing what this pull request does}}

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   resolves #3166 

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

### CODE REVIEW

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/repl.txt [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    --------
+    > function fcn( x, y, z ) { return x + y + z };
+    > var x = [ 1.0 ];
+    > var y = [ [ 2.0, 2.0 ] , [ 3.0, 3.0 ] ];
+    > var z = [ [ [ [ [ 2.0 ] , [ 3.0 ] ] ] ] ];
+    > var out = [ [ [ [ [ 0.0 ,0.0 ], [ 0.0 ,0.0 ] ] ] ] ];
+    > var shapes =[ [ 1 ], [ 2, 2 ], [ 1, 1, 1, 2, 1 ], [ 1, 1, 1, 2, 2 ] ];
+    > {{alias}}( [ x, y, z, out ], shapes, fcn );
+    > out
+    [ [ [ [ [ 5.0 ,5.0 ], [ 7.0 ,7.0 ] ] ] ] ]
```
Comment (kgryte) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/repl.txt]: @MeastroZI Same comment. I suggest doing a line-by-line review of your PR to ensure consistent formatting.

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/README.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+Applies a ternary callback to elements in three [broadcasted][@stdlib/array/base/broadcast-array] nested input arrays and assigns results to elements in a five-dimensional nested output array.
+
+```javascript
+var add = require( '@stdlib/math/base/ops/add3' );
+var zeros5d = require( '@stdlib/array/base/zeros5d' );
+
+var x = [ [ [ 1.0, 2.0 ] ] ];
+var y = [ [ [ 3.0 ], [ 4.0 ] ] ];
+var z = [ [ [ [4.0, 5.0] ], [ [4.0, 5.0] ] ] ];
```
Comment (Vinit-Pandit) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/README.md]:
```suggestion
var z = [ [ [ [ 4.0, 5.0 ] ], [ [ 4.0, 5.0 ] ] ] ];
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/benchmark/benchmark.js [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+			}
+		}
+		b.toc();
+
+		i4 = i % shapes[ 1 ][ 0 ];
+		i3 = i % shapes[ 1 ][ 1 ];
+		i2 = i % shapes[ 1 ][ 2 ];
+		i1 = i % shapes[1 ][ 3 ];
+		i0 = i % shapes[ 1 ][ 4 ];
+		if (isnan(arrays[ 3 ][ i4 ][ i3 ][ i2 ][ i1 ][ i0 ])) {
```
Comment (Vinit-Pandit) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/benchmark/benchmark.js]:
```suggestion
		if ( isnan(arrays[ 3 ][ i4 ][ i3 ][ i2 ][ i1 ][ i0 ] ) ) {
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/types/test.ts [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+{
+	const x = [ [ [ [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ];
+	const y = [ [ [ [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ];
+	const z = [ [ [ [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ];
+	const out = [ [ [ [ [ 0.0, 0.0 ], [ 0.0, 0.0 ] ] ] ] ];
+
+	const shapes: [ Array<number>, Array<number>, Array<number> ] = [ [ 1, 1, 1, 2, 2 ], [ 1, 1, 1, 2, 2 ], [ 1, 1, 1, 2, 2 ] ];
+
+	bternary5d( [ x, y, z, out ], shapes, fcn ); // $ExpectType void
+	bternary5d( [ x[0][0][0][0], y, z ], [ [ shapes[0][4] ], shapes[1], shapes[2] ], fcn ); // $ExpectType void
```
Comment (Vinit-Pandit) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/types/test.ts]:
```suggestion
	bternary5d( [ x[ 0 ][ 0 ][ 0 ][ 0 ], y, z ], [ [ shapes[ 0 ][ 4 ] ], shapes[ 1 ], shapes[ 2 ] ], fcn ); // $ExpectType void
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/types/test.ts [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+	const z = [ [ [ [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ];
+	const out = [ [ [ [ [ 0.0, 0.0 ], [ 0.0, 0.0 ] ] ] ] ];
+
+	const shapes: [ Array<number>, Array<number>, Array<number> ] = [ [ 1, 1, 1, 2, 2 ], [ 1, 1, 1, 2, 2 ], [ 1, 1, 1, 2, 2 ] ];
+	bternary5d( [ x, y, z, out ], shapes, 'abc' ); // $ExpectError
+	bternary5d( [ x, y, z, out ], shapes, 3.14 ); // $ExpectError
+	bternary5d( [ x, y, z, out ], shapes, true ); // $ExpectError
+	bternary5d( [ x, y, z, out ], shapes, false ); // $ExpectError
+	bternary5d( [ x, y, z, out ], shapes, null ); // $ExpectError
+	bternary5d( [ x, y, z, out ], shapes, ['1'] ); // $ExpectError
```
Comment (Vinit-Pandit) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/types/test.ts]:
```suggestion
	bternary5d( [ x, y, z, out ], shapes, [ '1' ] ); // $ExpectError
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/repl.txt [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+        Array shapes.
+
+    fcn: Function
+        Ternary callback.
+
+    Examples
+    --------
+    > function fcn( x, y, z ) { return x + y + z };
+    > var x = [ 1.0 ];
+    > var y = [ [ 2.0, 2.0 ] , [ 3.0, 3.0 ] ];
```
Comment (Vinit-Pandit) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/repl.txt]:
```suggestion
    > var y = [ [ 2.0, 2.0 ], [ 3.0, 3.0 ] ];
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/repl.txt (hunk 1 above)
Comment (Vinit-Pandit) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/repl.txt]:
```suggestion
    [ [ [ [ [ 5.0, 5.0 ], [ 7.0, 7.0 ] ] ] ] ]
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/repl.txt [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+    > var z = [ [ [ [ [ 2.0 ] , [ 3.0 ] ] ] ] ];
```
Comment (Vinit-Pandit) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/repl.txt]:
```suggestion
    > var z = [ [ [ [ [ 2.0 ], [ 3.0 ] ] ] ] ];
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/benchmark/benchmark.js [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var uniform = require( '@stdlib/random/base/uniform' ).factory;
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var pow = require( '@stdlib/math/base/special/pow' );
+var floor = require( '@stdlib/math/base/special/floor' );
+var add = require( '@stdlib/math/base/ops/add' );
```
Comment (kgryte) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/benchmark/benchmark.js]:
```suggestion
var add = require( '@stdlib/math/base/ops/add3' );
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/types/index.d.ts [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+*
+* var x = ones5d( shapes[ 0 ] );
+* var y = ones5d( shapes[ 1 ] );
+* var z = ones5d( shapes[ 2 ] );
+* var out = zeros5d( shapes[ 3 ] );
+*
+* bternary5d( [ x, y, z, out ], shapes, add );
+*
+* console.log( out );
+* // =>[ [ [ [ [ 3 ] ] ], [ [ [ 3 ] ] ] ], [ [ [ 3 ] ], [ [ [ 3] ] ] ] ]
```
Comment (kgryte) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/types/index.d.ts]: Decimals and spacing.

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/repl.txt [hunk 10]
```diff
@@ -0,0 +1,32 @@
...
+        Ternary callback.
+
+    Examples
+    --------
+    > function fcn( x, y, z ) { return x + y + z };
+    > var x = [ 1.0 ];
+    > var y = [ [ 2.0, 2.0 ], [ 3.0, 3.0 ] ];
+    > var z = [ [ [ [ [ 2.0 ], [ 3.0 ] ] ] ] ];
+    > var out = [ [ [ [ [ 0.0 ,0.0 ], [ 0.0 ,0.0 ] ] ] ] ];
+    > var shapes =[ [ 1 ], [ 2, 2 ], [ 1, 1, 1, 2, 1 ], [ 1, 1, 1, 2, 2 ] ];
```
Comment (kgryte) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/repl.txt]:
```suggestion
    > var shapes = [ [ 1 ], [ 2, 2 ], [ 1, 1, 1, 2, 1 ], [ 1, 1, 1, 2, 2 ] ];
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/benchmark/benchmark.js [hunk 11]
```diff
@@ -0,0 +1,136 @@
...
+			if ( isnan( arrays[ 3 ][ i4 ][ i3 ][ i2 ][ i1 ][ i0 ] ) ) {
+				b.fail( 'should not return NaN' );
+			}
+		}
+		b.toc();
+
+		i4 = i % shapes[ 1 ][ 0 ];
+		i3 = i % shapes[ 1 ][ 1 ];
+		i2 = i % shapes[ 1 ][ 2 ];
+		i1 = i % shapes[1 ][ 3 ];
```
Comment (kgryte) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/benchmark/benchmark.js]:
```suggestion
		i1 = i % shapes[ 1 ][ 3 ];
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/README.md [hunk 12]
```diff
@@ -0,0 +1,141 @@
...
+
+var shapes = [
+    [ 1, 1, 2 ],
+    [ 1, 2, 1 ],
+    [ 1, 2, 1, 2 ],
+    [ 1, 1, 2, 2, 2 ]
+];
+
+bternary5d( [ x, y, z, out ], shapes, add );
+// out => [ [ [ [ [ 8, 10 ], [ 9, 11 ] ], [ [ 8, 10 ], [ 9, 11 ] ] ] ] ]
```
Comment (kgryte) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/README.md]: Decimals.

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/types/test.ts [hunk 13]
```diff
@@ -0,0 +1,107 @@
...
+	const shapes: [ Array<number>, Array<number>, Array<number> ] = [ [ 1, 1, 1, 2, 2 ], [ 1, 1, 1, 2, 2 ], [ 1, 1, 1, 2, 2 ] ];
+
+	bternary5d( [ x, y, z, out ], shapes, fcn ); // $ExpectType void
+	bternary5d( [ x[ 0 ][ 0 ][ 0 ][ 0 ], y, z ], [ [ shapes[ 0 ][ 4 ] ], shapes[ 1 ], shapes[ 2 ] ], fcn ); // $ExpectType void
+}
+
+// The compiler throws an error if the function is provided a first argument which is not an array of nested arrays...
+{
+	const shapes: [ Array<number>, Array<number>, Array<number> ] = [ [ 1, 1, 1, 2, 2 ], [ 1, 1, 1, 2, 2 ], [ 1, 1, 1, 2, 2 ] ];
+	bternary5d( 'abc', shapes, fcn ); // $ExpectError
```
Comment (kgryte) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/types/test.ts:53]:
```suggestion

	bternary5d( 'abc', shapes, fcn ); // $ExpectError
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/types/test.ts [hunk 14]
```diff
@@ -0,0 +1,107 @@
...
+
+// The compiler throws an error if the function is provided a third argument which is not a valid callback...
+{
+	const x = [ [ [ [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ];
+	const y = [ [ [ [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ];
+	const z = [ [ [ [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ];
+	const out = [ [ [ [ [ 0.0, 0.0 ], [ 0.0, 0.0 ] ] ] ] ];
+
+	const shapes: [ Array<number>, Array<number>, Array<number> ] = [ [ 1, 1, 1, 2, 2 ], [ 1, 1, 1, 2, 2 ], [ 1, 1, 1, 2, 2 ] ];
+	bternary5d( [ x, y, z, out ], shapes, 'abc' ); // $ExpectError
```
Comment (kgryte) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/types/test.ts:89]:
```suggestion

	bternary5d( [ x, y, z, out ], shapes, 'abc' ); // $ExpectError
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/types/test.ts [hunk 15]
```diff
@@ -0,0 +1,107 @@
...
+
+// The compiler throws an error if the function is provided an unsupported number of arguments...
+{
+	const x = [ [ [ [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ];
+	const y = [ [ [ [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ];
+	const z = [ [ [ [ [ 1.0, 2.0 ], [ 3.0, 4.0 ] ] ] ] ];
+	const out = [ [ [ [ [ 0.0, 0.0 ], [ 0.0, 0.0 ] ] ] ] ];
+
+	const shapes: [ Array<number>, Array<number>, Array<number> ] = [ [ 1, 1, 1, 2, 2 ], [ 1, 1, 1, 2, 2 ], [ 1, 1, 1, 2, 2 ] ];
+	bternary5d(); // $ExpectError
```
Comment (kgryte) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/types/test.ts:107]:
```suggestion

	bternary5d(); // $ExpectError
```

Consistency in the use of whitespace based on L43.

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/lib/index.js [hunk 16]
```diff
@@ -0,0 +1,56 @@
...
+*
+* var x = ones5d( shapes[ 0 ] );
+* var y = ones5d( shapes[ 1 ] );
+* var z = ones5d( shapes[ 2 ] );
+* var out = zeros5d( shapes[ 3 ] );
+*
+* bternary5d( [ x, y, z, out ], shapes, add );
+*
+* console.log( out );
+* // =>[ [ [ [ [ 3 ] ] ], [ [ [ 3 ] ] ] ], [ [ [ [ 3 ] ] ], [ [ [ 3 ] ] ] ] ]
```
Comment (kgryte) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/lib/index.js]: Decimals.

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/lib/main.js [hunk 17]
```diff
@@ -0,0 +1,225 @@
...
+*
+* var x = ones5d( shapes[ 0 ] );
+* var y = ones5d( shapes[ 1 ] );
+* var z = ones5d( shapes[ 2 ] );
+* var out = zeros5d( shapes[ 3 ] );
+*
+* bternary5d( [ x, y, z, out ], shapes, add );
+*
+* console.log( out );
+* // =>[ [ [ [ [ 3 ] ] ], [ [ [ 3 ] ] ] ], [ [ [ [ 3 ] ] ], [ [ [ 3 ] ] ] ] ]
```
Comment (kgryte) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/lib/main.js]: Decimals.

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/lib/main.js [hunk 18]
```diff
@@ -0,0 +1,225 @@
...
+	S0 = sh[ 4 ];
+	S1 = sh[ 3 ];
+	S2 = sh[ 2 ];
+	S3 = sh[ 1 ];
+	S4 = sh[ 0 ];
+	if ( S0 <= 0 || S1 <= 0 || S2 <= 0 || S3 <= 0 || S4 <= 0 ) {
+		return;
+	}
+	o = broadcastArray( arrays[ 0 ], shapes[ 0 ], sh );
+
```
Comment (kgryte) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/lib/main.js]:
```suggestion
```

Consistent use of whitespace based on L143.

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/package.json [hunk 19]
```diff
@@ -0,0 +1,67 @@
...
+    "windows"
+  ],
+  "keywords": [
+    "stdlib",
+    "base",
+    "array",
+    "multidimensional",
+    "ndarray",
+    "matrix",
+    "3d",
```
Comment (kgryte) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/package.json]:
```suggestion
    "5d",
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/test/test.js [hunk 20]
```diff
@@ -0,0 +1,331 @@
...
+		]
+	];
+	bternary5d( [ x, y, z, w ], shapes, add );
+	t.deepEqual( w, expected, 'returns expected value' );
+
+	t.end();
+} );
+
+// ( And so on for the remaining parts of your code... )
+
```
Comment (kgryte) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/test/test.js]: What is this comment?

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/lib/main.js (hunk 17 above)
Comment (kgryte) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/lib/main.js]:
```suggestion
* // => [ [ [ [ [ 3 ] ] ], [ [ [ 3 ] ] ] ], [ [ [ [ 3 ] ] ], [ [ [ 3 ] ] ] ] ]
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/lib/index.js (hunk 16 above)
Comment (kgryte) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/lib/index.js]:
```suggestion
* // => [ [ [ [ [ 3 ] ] ], [ [ [ 3 ] ] ] ], [ [ [ [ 3 ] ] ], [ [ [ 3 ] ] ] ] ]
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/types/index.d.ts (hunk 9 above)
Comment (kgryte) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/types/index.d.ts]:
```suggestion
* // => [ [ [ [ [ 3 ] ] ], [ [ [ 3 ] ] ] ], [ [ [ 3 ] ], [ [ [ 3] ] ] ] ]
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/test/test.js [hunk 21]
```diff
@@ -0,0 +1,331 @@
...
+			[ [ [ 3.0 ] ], [ [ 3.0 ] ] ]
+		]
+	];
+	bternary5d( [ x, y, z, w ], shapes, add );
+	t.deepEqual( w, expected, 'returns expected value' );
+
+	t.end();
+} );
+
+// ( And so on for the remaining parts of your code... )
```
Comment (Vinit-Pandit) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/test/test.js]:
```suggestion
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/lib/main.js (hunk 17 above)
Comment (Vinit-Pandit) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/lib/main.js]:
```suggestion
* // =>[ [ [ [ [ 3.0 ] ] ], [ [ [ 3.0 ] ] ] ], [ [ [ [ 3.0 ] ] ], [ [ [ 3.0 ] ] ] ] ]
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/lib/index.js (hunk 16 above)
Comment (Vinit-Pandit) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/lib/index.js]:
```suggestion
* // =>[ [ [ [ [ 3.0 ] ] ], [ [ [ 3.0 ] ] ] ], [ [ [ [ 3.0 ] ] ], [ [ [ 3.0 ] ] ] ] ]
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/types/index.d.ts (hunk 9 above)
Comment (Vinit-Pandit) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/docs/types/index.d.ts]:
```suggestion
* // =>[ [ [ [ [ 3.0 ] ] ], [ [ [ 3.0 ] ] ] ], [ [ [ 3.0 ] ], [ [ [ 3.0 ] ] ] ] ]
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/README.md (hunk 12 above)
Comment (Vinit-Pandit) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/README.md]:
```suggestion
// out => [ [ [ [ [ 8.0, 10.0 ], [ 9.0, 11.0 ] ], [ [ 8.0, 10.0 ], [ 9.0, 11.0 ] ] ] ] ]
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/lib/index.js [hunk 22]
```diff
@@ -0,0 +1,56 @@
...
+*
+* var x = ones5d( shapes[ 0 ] );
+* var y = ones5d( shapes[ 1 ] );
+* var z = ones5d( shapes[ 2 ] );
+* var out = zeros5d( shapes[ 3 ] );
+*
+* bternary5d( [ x, y, z, out ], shapes, add );
+*
+* console.log( out );
+* // => [ [ [ [ [ 3 ] ] ], [ [ [ 3 ] ] ] ], [ [ [ [ 3 ] ] ], [ [ [ 3 ] ] ] ] ]
```
Comment (Vinit-Pandit) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/lib/index.js]:
```suggestion
* // => [ [ [ [ [ 3.0 ] ] ], [ [ [ 3.0 ] ] ] ], [ [ [ [ 3.0 ] ] ], [ [ [ 3.0 ] ] ] ] ]
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/lib/main.js [hunk 23]
```diff
@@ -0,0 +1,224 @@
...
+*
+* var x = ones5d( shapes[ 0 ] );
+* var y = ones5d( shapes[ 1 ] );
+* var z = ones5d( shapes[ 2 ] );
+* var out = zeros5d( shapes[ 3 ] );
+*
+* bternary5d( [ x, y, z, out ], shapes, add );
+*
+* console.log( out );
+* // => [ [ [ [ [ 3 ] ] ], [ [ [ 3 ] ] ] ], [ [ [ [ 3 ] ] ], [ [ [ 3 ] ] ] ] ]
```
Comment (Vinit-Pandit) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/lib/main.js]:
```suggestion
* // => [ [ [ [ [ 3.0 ] ] ], [ [ [ 3.0 ] ] ] ], [ [ [ [ 3.0 ] ] ], [ [ [ 3.0 ] ] ] ] ]
```

File: lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/test/test.js [hunk 24]
```diff
@@ -0,0 +1,330 @@
...
+			[ [ [ 3.0 ] ], [ [ 3.0 ] ] ],
+			[ [ [ 3.0 ] ], [ [ 3.0 ] ] ]
+		]
+	];
+	bternary5d( [ x, y, z, w ], shapes, add );
+	t.deepEqual( w, expected, 'returns expected value' );
+
+	t.end();
+} );
+
```
Comment (Vinit-Pandit) [inline lib/node_modules/@stdlib/array/base/broadcasted-ternary5d/test/test.js:135]:
```suggestion
```

**Review by kgryte — APPROVED** [review]
LGTM.

**Pull request merged**